### PR TITLE
feat(terraform): Adding 2 new AWS policies

### PR DESCRIPTION
**Pull request by praveen-panw**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

This PR has 2 YAML-based checks with detailed description below

### 1. Checkov Title - Ensure access control lists for S3 buckets are disabled
 
<ins>PC Policy ID</ins> - cab6f1a5-f6c5-4caf-bf10-b0f3a29e90ea
 
<ins>PC Policy Title</ins> - AWS S3 bucket access control lists (ACLs) in use

<ins>Description</ins>

This policy identifies AWS S3 buckets which are using access control lists (ACLs). ACLs are legacy way to control access to S3 buckets. It is recommended to disable bucket ACL and instead use IAM policies or S3 bucket policies to manage access to your S3 buckets.
 
<ins>Compliance standards </ins> - 
 
`MLPS 2.0 (Level 2)` `Secure Controls Framework (SCF) - 2022.2.1` `Sarbanes Oxley Act (SOX)` `AWS Foundational Security Best Practices standard`
 
<ins>Remediation:</ins>
 
1. Login to the AWS Console
2. Navigate to the 'S3' service
3. Click on the 'S3' resource reported in the alert
4. Click on the 'Permissions' tab
5. Under 'Object Ownership' click 'Edit'
6. Select 'ACLs disabled (recommended)'
7. Click on 'Save changes'  

<ins>CLI Command:</ins>

`aws s3api put-bucket-ownership-controls --bucket ${resourceName} --ownership-controls '{"Rules":[{"ObjectOwnership":"BucketOwnerEnforced"}]}'  `

<ins>CLI Command Description:</ins>

This CLI command requires 's3:PutBucketOwnershipControls' permission. Successful execution will update the ACL OwnershipControls to 'BucketOwnerEnforced' for the S3 bucket

### 2. Checkov Title - Ensure MWAA environment is not publicly accessible

<ins>Description</ins>

This policy is designed to safeguard AWS Managed Workflows for Apache Airflow (MWAA) environments by ensuring that they are not publicly accessible. This policy aims to mitigate security risks and prevent unauthorized access to MWAA environments, thereby safeguarding sensitive data and critical workflows.
 
<ins>Remediation:</ins>
 
To secure your VPC on AWS MWAA; follow below URL:
https://docs.aws.amazon.com/mwaa/latest/userguide/vpc-security.html

### CODE REVIEW

File: checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+metadata:
+  id: "CKV2_AWS_67"
```
Comment (gruebel) [inline checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml:2]:
```suggestion
  id: "CKV2_AWS_65"
```
I didn't find any `CKV2_AWS_65` or do you have it somewhere in the pipeline already 😄

File: checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  name: "Ensure MWAA environment is not publicly accessible"
+  category: "GENERAL_SECURITY"
```
Comment (gruebel) [inline checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml:4]:
```suggestion
  category: "NETWORKING"
```

**Review by gruebel — COMMENTED** [review]
nice work

File: checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml (hunk 1 above)
Comment (praveen-panw) [reply checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml:2]: Uh oh, noted.. Will update it.. 😛

File: checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml (hunk 2 above)
Comment (praveen-panw) [reply checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml]: Done

File: checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml (hunk 1 above)
Comment (praveen-panw) [reply checkov/terraform/checks/graph_checks/aws/AWS_private_MWAA_environment.yaml]: Done ✅

**Review by gruebel — APPROVED** [review]
🏆

**Review by maxamel — APPROVED** [review]

**Review by SteveVaknin — APPROVED** [review]

**Pull request merged**